home.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shop-Ship/Home</title>
+    <link rel="stylesheet" href="https://garuda-ui.netlify.app/index.css">
+    <link rel="stylesheet" href="/home.css">
+    <script src="https://kit.fontawesome.com/108bfd668b.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="e-com-header">
+        <nav class="e-com-navbar">
+            ShopShip
+            <ul class="e-com-nav-items">
+            </ul>
+            <input type="text" class="search-input" placeholder="search">
+            <div class="e-com-social">
+                <div class="ecom-login-container">
+                    <button class="login-btn">Login</button>
+                </div>
+                <a href="" class="e-com-nav-icons anchor-style pointer">
+                    <i class="fas fa-heart heart-icon">
+                        <div class="ecom-badge">4</div>
+                    </i>
+                </a>
+                <a href="" class="e-com-nav-icons anchor-style pointer">
+                    <i class="fas fa-shopping-cart cart-icon">
+                        <div class="ecom-badge">5</div>
+                        <span class="cart-text">cart</span></i>
+                </a>
+            </div>
+        </nav>
+      </header>
+      <main class="ecom-main px-5 py-3">
+          <div class="home-categories">
+              <div class="categoies p-1">
+                  <a href="" class="category-links flex-col-center">
+                      <img src="/assets/IMG_20220205_233718.png" alt="" class="category-img">
+                      <p class="category-text p-1">Mobiles</p>
+                  </a>
+              </div>
+              <div class="categoies p-1">
+                <a href="" class="category-links flex-col-center">
+                    <img src="/assets/fashion.jpeg" alt="" class="category-img">
+                    <p class="category-text p-1">Fashion</p>
+                </a>
+            </div>
+            <div class="categoies p-1">
+                <a href="" class="category-links flex-col-center">
+                    <img src="/assets/headphone-1.jpeg" alt="" class="category-img">
```

Comment:
Here you have added alt but it should add information about the image. because if your img fails to load user can see this alt on screen and having a meaningful alt will help.

Reply:
ok understood